data/generate_test_data.md

```diff
@@ -0,0 +1,16 @@
+## Generate keypair and container image for unit and integration tests
+
```

Comment:
I think it might be worth adding some more instructions here, for example getting the user to log into their container registry then set a shell variable to their user account. e.g. `export cr_user="<your container reg account>`
then updating the skopeo commands to replace `user` with `cr_user`.
I'm not sure if this is supposed to support more that just Docker Hub either?

Reply:
Yes, we can add more detail info for the setup. It should support different registries, I tried with a local registry and Docker Hub, encrypted container image both works, but the server side may change the `dest-compress-format`.